unity-renderer/Assets/Scripts/MainScripts/DCL/KernelConfiguration/KernelConfigTypes.cs

```diff
@@ -8,6 +8,7 @@ public class Features
         public bool enableBuilderInWorld = false;
         public bool enableAvatarLODs = false;
         public bool enableTutorial = true;
+        public bool enablePeopleCounter = false;
 
         public bool Equals(Features other) { return enableBuilderInWorld == other?.enableBuilderInWorld; }
```

Comment:
Could you please update the `Equals` method? It looks like a lot of fields were added and nobody updated the `Equals`  😅

Reply:
Shouldn't `Features` be a struct? like all value objects should 🤔